dotCMS/src/main/java/com/dotmarketing/business/cache/provider/h22/H22HikariPool.java

```diff
@@ -18,12 +18,14 @@
 	final int dbNumber;
 	final String dbRoot;
 	final String database;
+	final int minPoolSize = Config.getIntProperty("cache.h22.db.poolsize.min", 20);
 	final int maxPoolSize = Config.getIntProperty("cache.h22.db.poolsize.max", 500);
+	final int idleTimeoutMs = Config.getIntProperty("cache.h22.db.poolsize.idle.timeout", 600000);
 	final int connectionTimeout = Config.getIntProperty("cache.h22.db.connection.timeout", 1000);
 	final int setLeakDetectionThreshold = Config.getIntProperty("cache.h22.db.leak.detection.timeout", 0);
 	final HikariDataSource datasource;
 	final String folderName;
-	boolean running = false;
+	AtomicBoolean running = new AtomicBoolean(false);

```

Comment:
Codacy found an issue: [Field running has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5918493)